brayns/utils/image/codecs/StbiHelper.cpp

```diff
@@ -44,17 +44,30 @@ class XmpXmlMetadataBuilder
 public:
     static std::string build(const ImageMetadata &metadata)
     {
-        auto xmlTitle = "<bbp:Title>" + metadata.title + "</bbp:Title>";
-        auto xmlDescr = "<bbp:Description>" + metadata.description + "</bbp:Description>";
-        auto xmlWhereUsed = "<bbp:WhereUsed>" + _serializeList(metadata.whereUsed) + "</bbp:WhereUsed>";
-        auto xmlKeywords = "<bbp:Keywords>" + _serializeList(metadata.keywords) + "</bbp:Keywords>";
+        auto xmlTitle = "<photoshop:Headline>" + metadata.title + "</photoshop:Headline>";
+        auto xmlDescr = "<dc:description>" + metadata.description + "</dc:description>";
+        auto xmlWhereUsed = "<Iptc4xmpExt:Event>" + metadata.whereUsed + "</Iptc4xmpExt:Event>";
+        auto xmlKeywords = "<dc:subject>" + _serializeList(metadata.keywords) + "</dc:subject>";
         return xmlTitle + xmlDescr + xmlWhereUsed + xmlKeywords;
     }
 
 private:
     static std::string _serializeList(const std::vector<std::string> &list)
     {
-        std::string result = "<rdf:Bag>";
+        constexpr std::size_t bagSchemaSize = 19; // <rdf:Bag></rdf:Bag>
+        constexpr std::size_t itemSchemaSize = 17; // <rdf:li></rdf:li>
```

Comment:
Could also be

`constexpr auto size = std::char_traits::length("<rdf:Bag></rdf:Bag>");`

Or if no constexpr needed:

`auto size = std::strlen("...");`